incubator/hnc/pkg/kubectl/set.go

```diff
@@ -0,0 +1,82 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubectl
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var setCmd = &cobra.Command{
+	Use:     "set",
+	Short:   "Allows setting namespace hierarchy",
+	Aliases: []string{"set"},
+	Args:    cobra.MinimumNArgs(1),
```

Comment:
Shouldn't this be "exact args?"

Reply:
Yeah, actually that's correct. I'll change it.